proto/delivery/delivery.proto

```diff
@@ -57,9 +60,7 @@ message Request {
   // Optional.
   string search_query = 10;
 
-  // Optional. Number of Insertions to return.
-  // DEPRECATED: use paging intead.
-  int32 limit = 15;
+  reserved 15; // Deprecated
```

Comment:
how'd this // Deprecated come in?  we shouldn't have to explain reserved fields.